app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<section class="auth-form">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: false }) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :full_name %><br />
+      <%= f.text_field :full_name, autocomplete: "name" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
```

Comment:
- [ ] I think it would be great if you made the inputs the same as the given design as mentioned by the previous reviewer since you forgot to fix it so kindly add the submit button above in the nav and the inputs close to each other as shown ♻🔥🙏🏻🚀

- Your design
![image](https://github.com/gilberthappi/RoR_budget_App/assets/58553711/1f1f1d53-f401-4afb-9824-f25fc9ed5be1)

- Given
![image](https://github.com/gilberthappi/RoR_budget_App/assets/58553711/8aad6074-eb48-4f0d-aa08-67f38301669f)

Reply:
Thanks
<img width="157" alt="signup2" src="https://github.com/gilberthappi/RoR_budget_App/assets/110288182/c2cb1b5c-9bcb-4f1e-8daa-b9c1d6cbb880">

Reply:
Fixed.

Reply:
yes

Reply:
but @gilberthappi kindly deploy the app and provide the link on the readme file and also adjust login page design also according design guidelines

Reply:
yes Sir , but i need your help in my deployment, i think i did it i do not know what happened
may i show you

Reply:
please,
 i tried many times and render said, deploy live

Reply:
@gilberthappi i contact you on slack.

Reply:
okey

Reply:
@gilberthappi Good job for fixing signup and login page design according to design guidelines and mentioned by the previous reviewer and deploy the app using render and provide the link to ReadMe file.

- Signup page 
![image](https://github.com/gilberthappi/RoR_budget_App/assets/93598189/682a511b-d591-48de-b1f5-15553483852a)

- Login page 
![image](https://github.com/gilberthappi/RoR_budget_App/assets/93598189/43ba5dba-b30c-4120-9085-48a99460fa66)

- App deploy on the Render 
![image](https://github.com/gilberthappi/RoR_budget_App/assets/93598189/dd1a01e0-23d0-4bb9-81f9-68e72cd71777)